packages/noya-state/src/exportOptions.ts

```diff
@@ -0,0 +1,40 @@
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+
+export type ExportScale = {
+  scale: number;
+  absoluteSize: number;
+  visibleScaleType: Sketch.VisibleScaleType;
+};
+
+export type ExportSize = {
+  size: number;
+  visibleScaleType: Sketch.VisibleScaleType;
+};
+
+export function parseScale(scaleText: string): ExportSize | undefined {
+  const size = isNaN(parseFloat(scaleText))
+    ? parseFloat(scaleText.slice(0, -1))
+    : parseFloat(scaleText);
+
+  if (isNaN(size) && size > 0) return undefined;
```

Comment:
Just noticed this needs to be `||` and slightly different logic, cause a number can't be both NaN and greater than 0. I'll update it after I merge

Reply:
👁️ I didn't notice tht one !